xAPI.md

```diff
@@ -2996,7 +2998,9 @@ on the AP, and have the LRS query that API for the list of updated (or new)
 Statements periodically. This will likely only be a realistic option for systems 
 that provide a lot of data to the LRS.  
 
-Returns: ```200 OK```, Array of Statement id(s) (UUID).  
+**Content:** An array of Statements or a single Statement object to be stored. 
```

Comment:
It should either be "array of Statement objects" or "a single Statement to be stored", not the mixture of nomenclature it is now. I'm fine either way.